src/getweather.js

```diff
@@ -789,13 +808,16 @@ export async function getWeatherInfo(extension, gettext)
 
         let response;
         let forecastResponse;
+        let dailyResponse;
         try
         {
           let cur = loadJsonAsync("https://api.openweathermap.org/data/2.5/weather", params);
           let fore = loadJsonAsync("https://api.openweathermap.org/data/2.5/forecast", params);
-          let allResp = await Promise.all([ cur, fore ]);
+          let daily = loadJsonAsync("https://api.openweathermap.org/data/2.5/forecast/daily", params);
+          let allResp = await Promise.all([ cur, fore, daily]);
```

Comment:
Seems like this should either be,

```suggestion
          let allResp = await Promise.all([ cur, fore, daily ]);
```
or
```suggestion
          let allResp = await Promise.all([cur, fore, daily]);
```